tools/rp_storage_tool/src/bucket_reader.rs

```diff
@@ -889,11 +896,11 @@ impl BucketReader {
         let mut discovered_objects: Vec<ObjectMeta> = vec![];
 
         for (ntpr, partition_objects) in &mut self.partitions {
```

Comment:
could we rename this to `ntpr` to `lntpr` or similar?

Reply:
Done